testdata/project-v4-with-plugins/dist/chart/templates/certmanager/certificate.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   selfSigned: {}
+{{- end }}
 {{- if .Values.webhook.enable }}
```

Comment:
certs for webhooks must be created  if webhook and certmanager are enabled
certs for metrics must be created if metrics and certmanager are enabled

Reply:
It does but wasn't intentional. I guess I messed it up when I forced push.
The "problematic" line is this one  https://github.com/kubernetes-sigs/kubebuilder/blob/ad4afdb7c88a8d50883af248435eb1526f6f2de9/testdata/project-v4-with-plugins/dist/chart/templates/certmanager/certificate.yaml#L36 
The if with "**_.Values.certmanager.enable_**" is redundant as the value is already true on line 1.

I can either remove that extra condition on the an clause or have 3 different if statements.

Reply:
So we have no problem at all?
If we have I am ok with both
Because we need asap before the release change the webhook implementation
see if we can only provide webhooks and enable true instead of adding the values there.

Otherwise, we might need to just comment on the changes and say that webhook is not supported yet.
it might a better approach than provide something that we need to change soon